README.md

```diff
@@ -399,64 +140,97 @@ render() {
+| [**slidesPerScroll**](https://brainhubeu.github.io/react-carousel/docs/examples/slidesPerScroll) | *Number* | Number by which value will change on scroll (autoPlay, arrow click, drag)|
+| **stopAutoPlayOnHover** | *Boolean* | Determines if autoPlay should stop when mouse hover over carousel |
+| [**value**](https://brainhubeu.github.io/react-carousel/docs/examples/controlled) | *Number* | Current slide's index (zero based, depends on the elements order) |
+
+### Dots props
+
+| Prop | Type | Description |
+| --- | --- | --- |
+| [**number**](https://brainhubeu.github.io/react-carousel/docs/examples/dots) | *Number* | Number of slides in carousel you want to control. By default it will be set to the number of thumbnails (if provided) |
+| [**onChange**](https://brainhubeu.github.io/react-carousel/docs/examples/dots) | *Function* | OnChange callback (works the same way as onChange in Carousel component) |
+| **rtl** | *Boolean* | Indicating if the dots should have direction from Right to Left |
+| [**thumbnails**](https://brainhubeu.github.io/react-carousel/docs/examples/thumbnails) | *Array of ReactElements* | Array of thumbnails to show. If not provided, default dots will be shown |
+| [**value**](https://brainhubeu.github.io/react-carousel/docs/examples/dots) | *Number* | Current Carousel value |
+
+## Tests
+
+### Unit tests
 ```
 yarn test:unit
 ```
 
-## E2E tests
-
+### E2E tests
 ```
 yarn test:e2e
 ```
 
-## Local running
-In order to run the docs/ demo locally:
-- `cd docs-www`
-- if you want to connect demo with the carousel source code, replace `__RC_ENV__` into `development` in https://github.com/brainhubeu/react-carousel/blob/master/docs-www/src/globalReferences.js#L2 and remove the `.babelrc` file in the root directory; otherwise, it will use the carousel code installed in `docs-www/node_modules`
-- `yarn develop`
+## Contributing
+[The GitHub issues list](https://github.com/brainhubeu/react-carousel/issues) is our roadmap.
+You're more than welcome to vote:
+- with 👍if you like a given feature request or you'd like a given bug to be fixed
+- with ❤️ if you love a given feature request or fixing a given bug is critical for you
+- with 👎if in your opinion, a given feature would create more damages than the value provided by it or you consider a given bug to be a feature
+
+We don't give any guarantee to fix even the most liked issues but 👍and ❤️ increase probability of fixing while 👎decreases the probability of fixing.
+
+You're also more than welcome to:
+- submit a feature request
+- report a bug
+- ask a question
+- comment an issue, discussing the details
+- open a PR, fixing a given issue
+
+## Decision log
+
+### SSR
+When using `@brainhubeu/react-carousel` with SSR (Server-side Rendering), we recommend [Next.js](https://github.com/zeit/next.js) as `@brainhubeu/react-carousel` currently doesn't work on the server side so it must be rendered on the client side (maybe we'll provide server-side working in the future).
```

Comment:
other decisions:
- `cypress` instead of `gemini`/`hermione`
- `renovate` instead of `greenkeeper`
- deploying a test environment for each PR to `beghp`
- IssueHunt?

Reply:
We decided to move it to separate ticket